Jenkinsfile

```diff
@@ -61,14 +63,14 @@ node('docker') {
                             }
 
                             stage('Generate k8s Resources') {
-                                make 'create-temporary-release-resources'
-                                archiveArtifacts 'target/make/k8s/*.yaml'
+                                make 'helm-generate'
+                                archiveArtifacts 'target/k8s/**/*'
```

Comment:
Since you already defined a variable `helmTargetDir`, maybe you want to use it here?